packages/react-components/lib/tasks/create-task.tsx

```diff
@@ -786,6 +795,22 @@ export function CreateTaskForm({
     }
   };
 
+  const handlePriorityClick = () => {
+    setTaskRequest((prev) => {
+      if (!prev.priority || !parseTaskPriority(prev.priority)) {
+        return {
+          ...prev,
+          priority: createTaskPriority(true),
+        };
+      }
+
+      return {
+        ...prev,
+        priority: createTaskPriority(false),
+      };
+    });
```

Comment:
I was trying to avoid having the helper function accept `null` or `undefined`, I guess refactoring to what you suggested is cleaner, thanks https://github.com/open-rmf/rmf-web/pull/1000/commits/a6a4e41019c0074695f6fb1f87df51b661ef290a